server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -223,7 +223,14 @@ private Counts(final List<NodeInfo> nodeInfos) {
                     roles.merge(COORDINATING_ONLY, 1, Integer::sum);
                 } else {
                     for (DiscoveryNodeRole role : nodeInfo.getNode().getRoles()) {
-                        roles.merge(role.roleName(), 1, Integer::sum);
+                        // TODO: Remove the 'if' condition and only keep the statement in 'else' after removing MASTER_ROLE.
+                        // As of 2.0, CLUSTER_MANAGER_ROLE is added, and it should be taken as MASTER_ROLE
+                        if (role.equals(DiscoveryNodeRole.MASTER_ROLE) || role.equals(DiscoveryNodeRole.CLUSTER_MANAGER_ROLE)) {
```

Comment:
Sorry @tlfeng , afterthought but please feel free to skip it over, I see we do these checks a number of times, may be we could add a method to `DiscoveryNodeRole`:

```
boolean isClusterManager() {
    return this.equals(DiscoveryNodeRole.MASTER_ROLE) || this.equals(DiscoveryNodeRole.CLUSTER_MANAGER_ROLE);
}
```

May simplify quite a long `if` conditions ...

Reply:
Hi @reta , it's never late to provide your suggestion! I'm not in a hurry, so will accept any improvement. Thank you for your feedback! 👍👍 I will make the change.

Reply:
Updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2424/commits/a2661a83180507967e63f964e6b521b127bf0bff. The code change looks clearer now 😄 without many `||`.